src/Education.UnitTests/EducationTests.cs

```diff
@@ -22,6 +22,7 @@
 using Moq;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Education.Commands;
+using SonarLint.VisualStudio.Education.XamlGenerator;
```

Comment:
I moved the generator interface into a subfolder (because I expect the implement to required multiple separate class files) -> namespace changed.